packages/rum/src/resourceUtils.ts

```diff
@@ -39,54 +39,24 @@ export function computeResourceKind(timing: PerformanceResourceTiming) {
   return ResourceKind.OTHER
 }
 
-const durationPairs: Array<[keyof PerformanceResourceTiming, keyof PerformanceResourceTiming]> = [
-  ['connectStart', 'connectEnd'],
-  ['domainLookupStart', 'domainLookupEnd'],
-  ['responseStart', 'responseEnd'],
-  ['requestStart', 'responseStart'],
-  ['redirectStart', 'redirectEnd'],
-  ['secureConnectionStart', 'connectEnd'],
-]
-function reporteAbnormalEntry(entry: PerformanceResourceTiming) {
-  let error = ''
-  for (const [start, end] of durationPairs) {
-    if (entry[start] < 0) {
-      error += `${start} is negative\n`
-    } else if (entry[start] > entry[end]) {
-      error += `${start} is greater than ${end}\n`
-    }
-  }
-  if (error) {
-    addMonitoringMessage(`Got an abnormal PerformanceResourceTiming:
-${error}
-Entry: ${JSON.stringify(entry)}`)
+function formatTiming(start: number, end: number) {
+  if (start <= 0 || end <= 0 || end < start) {
+    return undefined
   }
+  return { duration: msToNs(end - start), start: msToNs(start) }
 }
 
 export function computePerformanceResourceDetails(
   entry?: PerformanceResourceTiming
 ): PerformanceResourceDetails | undefined {
   if (entry && hasTimingAllowedAttributes(entry)) {
-    reporteAbnormalEntry(entry)
     return {
-      connect: { duration: msToNs(entry.connectEnd - entry.connectStart), start: msToNs(entry.connectStart) },
-      dns: {
-        duration: msToNs(entry.domainLookupEnd - entry.domainLookupStart),
-        start: msToNs(entry.domainLookupStart),
-      },
-      download: { duration: msToNs(entry.responseEnd - entry.responseStart), start: msToNs(entry.responseStart) },
-      firstByte: { duration: msToNs(entry.responseStart - entry.requestStart), start: msToNs(entry.requestStart) },
-      redirect:
-        entry.redirectStart > 0
-          ? { duration: msToNs(entry.redirectEnd - entry.redirectStart), start: msToNs(entry.redirectStart) }
-          : undefined,
-      ssl:
-        entry.secureConnectionStart > 0
-          ? {
-              duration: msToNs(entry.connectEnd - entry.secureConnectionStart),
-              start: msToNs(entry.secureConnectionStart),
-            }
-          : undefined,
+      connect: formatTiming(entry.connectStart, entry.connectEnd),
+      dns: formatTiming(entry.domainLookupStart, entry.domainLookupEnd),
+      download: formatTiming(entry.responseStart, entry.responseEnd),
+      firstByte: formatTiming(entry.requestStart, entry.responseStart),
+      redirect: formatTiming(entry.redirectStart, entry.redirectEnd),
+      ssl: formatTiming(entry.secureConnectionStart, entry.connectEnd),
```

Comment:
what about discarding the performance entry instead of only the wrong timing?
Not sure that it worth handling incomplete entries on the app